README.md

```diff
@@ -21,6 +21,16 @@ $ npm install --save react-native-languages
 $ yarn add react-native-languages
 ```
 
+####
+If you are using this lib as shown in `example` folder, you have to install `i18n-js` lib:
```

Comment:
This is to specific. I don't encourage the usage of `i18n-js`, especially like this (the way it was used in `react-native-i18n`.

It's a very basic and unperformant way (no memoization) of doing i18n. I should provides multiples examples with differents libraries instead.